styles.css

```diff
@@ -0,0 +1,114 @@
+/*        O__O
+          (oo)
+   /-------\/
+  / |     ||
+ *  ||----||
+    ~~    ~~
+Cow at Disneyland
+*/
+
+* {
+	font-family: din-condensed, sans-serif;
+	font-weight: 400;
+	font-style: normal;
+}
+
+body {
+	margin: 0;
+}
+
+/*NAVBAR STYLES*/
+
+nav {
+	overflow: auto;
+	background-color: #ffffff;
+	color: #000000;
+	font-size: 25px;
+	height: 70px;
+	}
+
+nav #logo {
+	float: left;
+	font-size: 40px;
+	padding: 8px;
+	font-weight: 600;
+	margin-left: 30px;
+	margin-top: 8px;
+}
+
+#hamburger {
+	float: right;
+	display: none;
+}
+
+.hamburgerRow {
+  width: 30px;
+  height: 4px;
+  background-color: black;
+  margin: 6px 0;
+}
```

Comment:
these four lines aren't indented the way the others are, if that bothers you

Reply:
thanks for pointing that out